lib/perl/Genome/WorkflowBuilder/Command.pm

```diff
@@ -99,6 +99,13 @@ sub _get_ptero_execute_method {
     $ptero_lsf_parameters->{environment} = $self->_get_sanitized_env();
     $ptero_lsf_parameters->{user} = Genome::Sys->username;
     $ptero_lsf_parameters->{cwd} = Cwd::getcwd;
+    $ptero_lsf_parameters->{pollingInterval} =
+        Genome::Config::get('ptero_lsf_polling_interval') + 0;
+
+    my $project_name = Genome::Config::get('lsf_project_name');
+    if ($project_name) {
+        $ptero_lsf_parameters->{options}{projectName} = $project_name;
```

Comment:
Properly handling the project name seems like it depends on #1129, which gets the project name from the Workflow XML.  Deciding between that XML and this environment variable might then go inside `_get_ptero_lsf_parameters`.